zerver/webhooks/jotform/doc.md

```diff
@@ -1,10 +1,14 @@
+# Zulip Jotform integration
+
 Get Zulip notifications for your Jotform responses!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
    By default, the integration will use the form's title as the topic.
```

Comment:
This line feels out of place

![image](https://github.com/zulip/zulip/assets/78212328/d3082b83-c19a-4c63-a565-bc77a4ec9570)